### PR TITLE
Remark: update UG and DG

### PR DESCRIPTION
**Pull request by alientian**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -397,23 +397,16 @@ An example of an error message is shown below.
...
-</div>
-
-<div markdown="block" class="alert alert-success">
-**:bulb: Useful Tips:**<br>
-* You can edit remarks through the edit or remark command!
-</div>
+**Examples:** 
 
-**Examples:** </br>
 1.You received an email from Microsoft to interview for a Frontend Developer role instead, which is different from the Software Engineer role you applied for.
```
Comment (papataco14) [inline docs/UserGuide.md:410]: Use the new formatting for this. change from 1. to **Step 1** (in bold).

File: docs/images/ModelClassDiagram.png
Comment (papataco14) [inline docs/images/ModelClassDiagram.png:1]: it seems that the labels "all" and "filtered" have been swapped for unique company list and model manager.

File: src/main/java/seedu/address/logic/commands/RemarkCommand.java [hunk 2]
```diff
@@ -32,7 +32,7 @@ public class RemarkCommand extends Command {
             + "Parameters: INDEX (must be a positive integer)\n"
             + "Example: " + COMMAND_WORD + " 1";
 
-    public static final String MESSAGE_NO_REMARK = "Remarks missing in entry.";
+    public static final String MESSAGE_NO_REMARK = "Oops! No remarks found in entry. Please try again!";
```
Comment (papataco14) [inline src/main/java/seedu/address/logic/commands/RemarkCommand.java:35]: try again for what

**Review by GlendaChong — APPROVED** [review]
LGTM

**Review by papataco14 — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/address/logic/commands/RemarkCommand.java (hunk 2 above)
Comment (alientian) [reply src/main/java/seedu/address/logic/commands/RemarkCommand.java:35]: its the standardised format for error msg

**Review by papataco14 — APPROVED** [review]

**Pull request merged**